src/dotnet/commands/dotnet-new/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.DotNet.Tools.New
+{
+    internal static class LocalizableStrings
+    {
+        public static const string AppName = "dotnet new";
+
```

Comment:
I don't want to change style subjectively. Distinct statements get a line in between to show they're not meaningfully related. Consistency avoids subjective fights over "I don't think these are related!"